lm/srilm.py

```diff
@@ -129,6 +133,7 @@ def __init__(
         self.ngram_order = ngram_order
         self.counts = counts
         self.vocab = vocab
+        self.data_for_opt = data_for_optimization
```

Comment:
```suggestion
        self.data_for_optimization= data_for_optimization
```
just for consistent naming